modules/processing/parsers/CAPE/Loki.py

```diff
@@ -171,9 +171,6 @@ def config(filebuf):
 
     cfg = {}
     urls = decoder(filebuf)
-    if urls:
-        cfg.setdefault("address", [])
-
```

Comment:
These lines were removed because `cfg["address"]` will always be created in both situations:

Situation 1: `urls` is not an empty list -> `cfg["address"]` will be overwritten by `cfg["address"] = [url.decode() for url in urls]`
Situation 2: `urls` is an empty list -> `cfg["address"] = [url.decode() for url in urls]` will still create an empty list `[]` for `cfg["address"]`

Reply:
make sense, thanks for comment